### PR TITLE
Added modern fedora path

### PR DESCRIPTION
**Pull request by darakian**
I've added the dnf package manager as an option. Works fine for me locally on fedora 23.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -42,6 +42,10 @@ if [[ "$unamestr" == 'Linux' ]]; then
     echo "installing pip"
     sudo pacman -S --needed --noconfirm base-devel python2-pip python2-virtualenv
     PIP="pip2"
+  elif [ $(which dnf) ]; then
+    sudo dnf install -y python-pip python-devel gcc gcc-c++ python-virtualenv glib2-devel
+    PIP="pip2"
+  fi
```
Comment (nedwill) [inline install.sh]: I think this is breaking Travis.

**Comment by nedwill** [comment]
There's a mistake, see the comment I made on the commit.

**Comment by darakian** [comment]
Give this one a check.

**Comment by nedwill** [comment]
Looks good, thanks for the contribution!

**Pull request merged**